modules/project/main.tf

```diff
@@ -0,0 +1,46 @@
+locals {
+  project_folder = coalesce(var.custom_project_folder, var.project_name)
+}
+
+module "workspace" {
+  source = "../workspaces"
+
+  count = var.enabled ? 1 : 0
+
+  auto_apply            = false // TODO: verify
+  file_triggers_enabled = true
+  name                  = "${var.environment}-${var.project_name}"
+  organization          = var.organization
+  trigger_prefixes      = ["${var.projects_path}/${local.project_folder}/*.tf"]
```

Comment:
feature flag `*.tf` so the user can change the files (e.g. maybe they have other filesin that folder that affect it).

Reply:
(default to `*.tf`)

Reply:
@osterman I went ahead and made this configurable at the project level via the YAML config. Let me know if that's what you had in mind 👍